happi/tests/test_client.py

```diff
@@ -316,10 +316,29 @@ def test_choices_for_field(happi_client: Client):
 
 def test_searchresults(client_with_three_valves: Client):
     valve1 = client_with_three_valves['VALVE1']
-    assert valve1.metadata == valve1
```

Comment:
Could this be brought back?

Reply:
This assert clause should fail, since the `valve1` and `valve1.metadata` aren't the same class.  Is this behavior something we want?

Reply:
Seems the original intent was: "did I get back what I requested?" Even a minimal check would suffice, even if not all the metadata